lib/routes.js

```diff
@@ -4,26 +4,26 @@ const routes = nextRoutes()
 routes
   .add('index', '/')
   .add('signin', '/anmelden')
-  .add('account', '/account')
+  .add('account', '/konto')
   .add('profile', '/~:slug')
-  .add('discussions', '/discussions')
-  .add('discussion', '/discussion')
-  .add('events', '/events/:slug?')
+  .add('discussion', '/diskussion')
+  .add('events', '/veranstaltungen')
+  .add('event', '/veranstaltung/:slug', 'events')
   .add('faq', '/faq')
   .add('feed', '/feed')
   .add('community', '/community')
   .add('crew', '/crew')
   .add('en', '/en')
   .add('manifest', '/manifest')
-  .add('media', '/media')
-  .add('pledge', '/pledge')
-  .add('claim', '/claim')
+  .add('media', '/medien')
+  .add('pledge', '/angebote')
+  .add('claim', '/abholen')
   .add('updates', '/updates/:slug?')
-  .add('notifications', '/notifications')
-  .add('legal/imprint', '/legal/imprint')
-  .add('legal/privacy', '/legal/privacy')
-  .add('legal/statute', '/legal/statute')
-  .add('legal/tos', '/legal/tos')
+  .add('notifications', '/notifikation')
```

Comment:
or maybe just `/meldung `to keep it short?

Reply:
I like to word but consider it too newsy to use here—«eine Republik Meldung» should just be a sign in confirmation^^

Reply:
maybe /mitteilung then, like the page title?